elastic/security/track.json

```diff
@@ -115,128 +107,43 @@
       "template": "./templates/component/.fleet_agent_id_verification-1.json",
       "template-path": "component_template"
     },
-    {
-      "name": ".fleet_component_template-1",
-      "template": "./templates/component/.fleet_component_template-1.json"
-    },
     {
       "name": ".fleet_globals-1",
       "template": "./templates/component/.fleet_globals-1.json",
       "template-path": "component_template"
     },
     {% endif %}
-    {
-      "name": "logs-endpoint.events.file@mappings",
-      "template": "./templates/component/logs-endpoint.events.file@mappings.json"
-    },
-    {% if p_skip_delete_component_template == false %}
     {
       "name": "logs-endpoint.events.file@package",
       "template": "./templates/component/logs-endpoint.events.file@package.json",
       "template-path": "component_template"
     },
-    {% endif %}
```

Comment:
@salvatore-campagna Can you comment on removal conditional based on `skip_delete_component_template` ex `skip_fleet_globals`? This was originally introduced in https://github.com/elastic/rally-tracks/pull/644. I'm not sure we need this at all. We are either using a vanilla cluster where we should be able to delete those component templates freely, or we use a setup where Kibana is installing those, and then we shouldn't touch those by skipping selected track tasks.

Reply:
As far as I recall now we should be ok with deleting them (not skipping delete).

Reply:
Ok, so let's remove it entirely, including this from `track.json`:
```
{% set p_skip_delete_component_template = (skip_delete_component_template | default(false) ) %}
```